cmd/node/config/config.toml

```diff
@@ -717,6 +717,8 @@
         CallGCWhenShuffleOut = true
         ExtraPrintsOnShuffleOut = true
         DoProfileOnShuffleOut = true
+    [Debug.Epoch]
```

Comment:
let's rephrase this. It is almost impossible to figure out what it does. Or the Enabled flag.

Reply:
fixed